agent/app/agent_capability.go

```diff
@@ -425,6 +433,18 @@ func (agent *ecsAgent) appendExecCapabilities(capabilities []*ecs.Attribute) ([]
 	return appendNameOnlyAttribute(capabilities, attributePrefix+capabilityExec), nil
 }
 
+func (agent *ecsAgent) appendServiceConenctCapabilities(capabilities []*ecs.Attribute) ([]*ecs.Attribute, error) {
+	if loaded, _ := agent.appNetLoader.IsLoaded(agent.dockerClient); !loaded {
+		_, err := agent.appNetLoader.LoadImage(agent.ctx, agent.dockerClient)
```

Comment:
Where do we start relay container? I think we want to verify that we are able to start relay and that it's healthy, before we append the capability.